packages/docs/content/components/inline-error.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Inline Error
+intro: Inline errors give validation feedback for specific form fields.
+core:
+  githubLink: design-system/src/components/InlineError
+  storybookLink: components-inlineerror--docs
+---
+
+## Examples
+
+<StorybookExample
+  componentName="inline-error"
+  sourceFilePath="components/InlineError/InlineError.tsx"
+  storyId="components-inlineerror--default"
+/>
+
+## Code
+
+### React
+
+<SeeStorybookForGuidance storyId={'components-inlineerror--docs'} />
+
+### Style customization
+
+The following CSS variables can be overridden to customize Form components:
+
+<ComponentThemeOptions componentname="form" />
+
+## Guidance
+
+- Inline errors are built in to all design-system form fields but can be used on their own to create custom fields.
+- Inline errors should sit either directly above or below the input, depending on the theme defaults.
+- Visually align inline validation messages with the input fields, so people using screen magnifiers can read them quickly.
+- The form field should have an `aria-describedby` attribute that references the `id` of the error message.
+
+#### Content
+
+- Describe what the user still needs to do in a mandatory field that has been left incomplete or unanswered,  or answered incorrectly.
+- Repeat what the form label is asking the user to do, but with slightly more detail.
+- Use sentence case.
+- End with a period.
+- Use the fewest words possible (i.e., don't use "please").
+
+## Learn More
+
+- [Form Guidelines](/patterns/Forms/forms/)
+
+## Component maturity
+
+<MaturityChecklist
+  a11yStandards={true}
+  color={true}
+  forcedColors={true}
+  screenReaders={true}
+  keyboardNavigable={null}
```

Comment:
same as hint comment, the children for this can be anything including interactive elements, so this should be true imo